InstantPrintTool.py

```diff
@@ -39,13 +40,15 @@ def __init__(self, iface, populateCompositionFz=None):
         self.rubberband = None
         self.oldrubberband = None
         self.pressPos = None
+        self.printer = QPrinter()
```

Comment:
Does this measurably slow down the QGIS start time? If yes, assign `self.printer = None` here and in `__print()` below assign it the `QPrinter` the first time it is called (i.e. if it is `None`).